include/zephyr/drivers/dma/dma_stm32.h

```diff
@@ -85,6 +85,10 @@
 #define STM32_DMA_FEATURES_FIFO_THRESHOLD(features)	0
 #endif
 
+#ifdef CONFIG_SOC_SERIES_STM32H7RSX
+#define __LL_DMA_GET_CHANNEL_INSTANCE LL_DMA_GET_CHANNEL_INSTANCE
+#endif /* CONFIG_SOC_SERIES_STM32H7RSX */
```

Comment:
Please avoid HAL symbols redefinitions but add the case for `STM32_DMA_GET_INSTANCE` definition below.